integration/chartconfig/chartconfig.go

```diff
@@ -0,0 +1,43 @@
+// +build k8srequired
+
+package chartconfig
+
+import (
+	"bytes"
+	"html/template"
+
+	"github.com/giantswarm/e2etemplates/pkg/e2etemplates"
+)
+
+type ChartConfigValues struct {
```

Comment:
It would be cool if `e2etemplates` was providing that struct. Otherwise we'll have to define it in every project that uses the template.

Reply:
I'll provide a PR to `e2etemplates` and we can discuss there.